src/app/auth/sign-up/sign-up.component.html

```diff
@@ -0,0 +1,42 @@
+<form [formGroup]="signUpForm"
+  (ngSubmit)="onSubmit()">
+  <mat-form-field>
+    <mat-label>Username</mat-label>
+    <input matInput
+      formControlName="displayName"
+      name="displayName"
+      type="text"
+      required />
+    <mat-error *ngIf="hasError('email', 'required')">Please enter your email</mat-error>
```

Comment:
do you check valid email for the displayed name?

Reply:
Fixed